mlflow/utils/request_utils.py

```diff
@@ -41,9 +42,10 @@ def augmented_raise_for_status(response):
 
 def download_chunk(range_start, range_end, headers, download_path, http_uri):
     combined_headers = {**headers, "Range": f"bytes={range_start}-{range_end}"}
+    timeout = DOWNLOAD_CHUNK_TIMEOUT_100M * (range_end - range_start) / 100_000_000
```

Comment:
Can `timeout` be float?

Reply:
10 minutes for 100MB seems very long. Can we use a smaller timeout?

Reply:
10minutes for 100MB = 170KB/s, what number do you prefer?

Reply:
1 minute?

Reply:
5 minutes sounds good.